test/test_unit_init_controller.py

```diff
@@ -116,7 +116,7 @@ def test_user_confirms_force_overwrites_writes_cert_to_file(self, mock_input, mo
     '''
     # The conjurrc file exists and the CLI prompts
     # if the user wants to overwrite
-    @patch('conjur.init.init_logic')
+    @patch('conjur.logic.init_logic')
```

Comment:
is this the only place this change in directory applies?

Reply:
There are many patches that changed becuase of the dir change